media-gfx/freecad/freecad-9999.ebuild

```diff
@@ -65,7 +65,7 @@ RESTRICT="mirror"
 #		salome-smesh - science overlay
 #		zipio++ - not in portage yet
 
-S="${WORKDIR}/FreeCAD-${PV}"
+S="${WORKDIR}/freecad-${PV}"
```

Comment:
can you just make this
`S=${WORKDIR}/${P,,}`
which is more succinct and a bash 4 feature